File: easy/distribution_of_sweets/distribution_of_sweets.java
package ejercicios;

import java.util.Scanner;

/**
 *06/05/2021
 * @author Andre Peralta
 */
public class metodo {
    public static void main(String[] args) {

        Scanner scan = new Scanner(System.in);
        
        System.out.println("Siblings : ");
        int siblings = scan.nextInt();
        
        System.out.println("Popsicles ");
        int popsicles = scan.nextInt();
        
        if(popsicles%siblings == 0){
            System.out.println("give away");
        }
        else{
            System.out.println("eat them yourself");
        }
   }
}

File: easy/distribution_of_sweets/distribution_of_sweets.rb
"""
@author	André Peralta
@version 20/05/2021
"""

def distribution_of_sweets siblings,popsicles

	if popsicles%siblings == 0
		return "give away"
	else
		return "eat them yourself"
	end
end


puts "Number of siblings: "
siblings = gets.to_i

puts "Number of popsicles: "
popsicles = gets.to_i

print distribution_of_sweets(siblings,popsicles)

